deploy.sh

```diff
@@ -12,12 +12,12 @@ sleep 3
 python /opt/cloudstack/tools/marvin/marvin/deployDataCenter.py -i /opt/zones.cfg
 
 export CLOUDSTACK_ENDPOINT=http://127.0.0.1:8096
-export CLOUDSTACK_KEY=""
-export CLOUDSTACK_SECRET=""
+export CLOUDSTACK_KEY=dummy
```

Comment:
@resmo I'm not sure if setting the values to `dummy` could break the tests?

Reply:
@rhtyd This is tested locally. 

The dummy values are necessary in the new version of cs. it won't allow to have empty values, otherwise it would look for a cloudstack.ini config file. 

the endpoint is the one used which doesn't need real authentication.